src/index.js

```diff
@@ -32,7 +32,16 @@ const students = [
     description1: "Senior &ndash; Computer Science @ Benedict College ",
     description2: "Software Engineer 👩🏾‍💻 ",
     dob: "1997-02-07"
+  },
+
+  {
+    name: "Ashanique Moore",
+    avatar: "public/student-avatars/ashanique-avatar.jpeg",
+    desription1: "Computer Science, Benedict College",
```

Comment:
Typo. That's why if you run the app you see `undefined`
![image](https://user-images.githubusercontent.com/10066805/215620770-33a095b1-cf24-426e-bdd6-e0c91f97f52f.png)